providers/aws/resources/aws_account.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 func (a *mqlAwsAccount) id() (string, error) {
-	return "aws.account/" + a.Id.Data, nil
```

Comment:
just a heads up, there are a couple other adjustments around discovery that we will need to do if this change goes in

Reply:
This change feels worth it IMO. The old output didn't really make sense. This output is better for asset inventory and customer usage. We should definitely understand the impact of this breaking change though anywhere else in the product.